Horion/Module/Modules/Bhop.cpp

```diff
@@ -36,4 +38,15 @@ void Bhop::onMove(C_MoveInputHandler* input) {
 	moveVec.y = player->velocity.y;
 	moveVec.z = moveVec2d.y * speed;
 	if(pressed) player->lerpMotion(moveVec);
+	if (counter == 2) {
+		counter = 0;
+	} else {
+		counter++;
+	}
+
+	if (counter == 1) {
```

Comment:
bruh...

Reply:
instead of this whole weird counter code you could just make use of a static bool to run something every second tick

Reply:
you could also do this
		if (player->onGround && pressed && !input->isJumping)
			player->velocity.y = height;